Xamarin.PropertyEditing.Windows/ByteToPercentageConverter.cs

```diff
@@ -10,21 +10,14 @@ namespace Xamarin.PropertyEditing.Windows
 	internal class ByteToPercentageConverter : MarkupExtension, IValueConverter
 	{
 		public object Convert (object value, Type targetType, object parameter, CultureInfo culture)
-		{
-			if (!(value is byte)) return DependencyProperty.UnsetValue;
-
-			var alpha = (byte)value;
-			return (alpha / 2.55d);
-		}
+			=> !(value is byte byteValue) ? DependencyProperty.UnsetValue
+				: byteValue / 2.55d;
 
 		public object ConvertBack (object value, Type targetType, object parameter, CultureInfo culture)
-		{
-			if (value is double doubleValue) {
-				if ((doubleValue < 0) || (doubleValue > 100)) return DependencyProperty.UnsetValue;
-				return System.Convert.ToByte (doubleValue * 2.55d);
-			}
-			return DependencyProperty.UnsetValue;
-		}
+			=> !(value is double doubleValue) ? DependencyProperty.UnsetValue
+				: doubleValue < 0 ? 0
+				: doubleValue > 100 ? 255
+				: System.Convert.ToByte (doubleValue * 2.55d);
```

Comment:
I find this being concise at the cost of readability.

Reply:
I don't know about that, I happen to find it more readable in fact: you clearly see each case and its outcome.